packages/next/client/components/reducer.ts

```diff
@@ -1151,7 +1152,7 @@ function clientReducer(
           pushRef: state.pushRef,
           // Apply focus and scroll.
           // TODO-APP: might need to disable this for Fast Refresh.
```

Comment:
```suggestion
```
these 2 lines aren't needed anymore ig?

Reply:
Still needed, needs to be verified. Please don't review random PRs.